### PR TITLE
UKEF Project Finance Page

### PR DESCRIPTION
**Pull request by erdem**
https://uktrade.atlassian.net/browse/XOT-812

### CODE REVIEW

File: core/sass/ukef-landing-page.scss [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      margin-top: 0.5em;
+    }
+  }
+
+  img{
+    width: 100%;
+  }
+
+  .heading-light{
+    font-family: $brand-font-alternative;
```
Comment (Kenneth-Siu) [inline core/sass/ukef-landing-page.scss]: Would it make sense to use an existing heading css class from directory-components and then overriding the font-family?

Also helps with responsiveness 👌

File: core/sass/ukef-landing-page.scss [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    font-weight: 700;
+    font-size: 36px;
+  }
+
+  .text{
+    font-size: 24px;
```
Comment (Kenneth-Siu) [inline core/sass/ukef-landing-page.scss]: Is it possible to just use one of the font classes from directory-components on these?

File: ukef/templates/ukef/landing_page.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+		</div>
+	</section>
+
+	<section id="why-we-offer-section" class="why-we-offer-section padding-top-60 background-stone-30">
+		<div class="container">
+			<div class="grid-row">
+				<div class="column-half padding-0">
+						<img src="{% static 'images/ukef_landing_offer_section_image.jpg' %}" id="offer-section-image " class="offer-section-image">
+				</div>
+				<div class="column-half offer-section-text padding-0">
```
Comment (Kenneth-Siu) [inline ukef/templates/ukef/landing_page.html]: Would `padding-horizontal-45 padding-vertical-0` make sense here?

File: core/sass/ukef-landing-page.scss [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+  //TABLET
+  @media screen and (max-width: 641px) {
+    .flag{
+      display: none;
+    }
+
+    .why-we-offer-section{
+      .offer-section-text{
+        padding: 30px;
```
Comment (Kenneth-Siu) [inline core/sass/ukef-landing-page.scss]: There's `padding-30-m` to apply directory-component styles at particular screen sizes (and larger)

File: core/sass/ukef-landing-page.scss [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+      span {
+        &:before {
+          background-color: $grey;
+          border-color: $grey;
+          color: $white;
+        }
+      }
+    }
+
+    .complete {
```
Comment (Kenneth-Siu) [inline core/sass/ukef-landing-page.scss]: Can't find this usages of this class either.

File: core/sass/ukef-landing-page.scss [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+      span {
+        &:before {
+          background-color: $grey;
+          border-color: $grey;
+          color: $grey;
+        }
+      }
+    }
+
+    .verbose {
```
Comment (Kenneth-Siu) [inline core/sass/ukef-landing-page.scss]: And can't find this usages of this class either.

File: ukef/templates/ukef/landing_page.html [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+					<div class="text column-eight-twelfths teaser-text">
+						UK Export Finance (UKEF) works with banks to provide ﬂexible, innovative, and competitive ﬁnancing solutions. We support UK exporters, international project sponsors and buyers of UK goods and services.
+					</div>
+				</div>
+				<div class="column-quarter more-about-section">
+					<div class="panel-vertical-narrow padding-top-0">
+						<h2 class="heading heading-small margin-top-15 margin-bottom-15">More about UKEF</h2>
+						<ul>
+							<li>
+								<h3 class="heading-small"><a href="#" class="link">Check our country cover</a></h3>
```
Comment (Kenneth-Siu) [inline ukef/templates/ukef/landing_page.html]: I assume all the dead links on this page are going to link to other places or something in the future, is that right?

File: core/sass/ukef-landing-page.scss (hunk 2 above)
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: Do you know the class name for the use?

File: core/sass/ukef-landing-page.scss [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+        left: -1.525em;
+        line-height: 1.6em;
+        position: absolute;
+        text-align: center;
+        top: 0;
+        width: 2.4em;
+      }
+    }
+
+    .current {
```
[the remark this answers is not in the document]
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: It could be, I'll check it.

File: ukef/templates/ukef/landing_page.html (hunk 7 above)
Comment (erdem) [reply ukef/templates/ukef/landing_page.html]: Yes, that's correct links will populate further pull-requests.

File: ukef/templates/ukef/landing_page.html [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+{% block head_title %}The UK’s Export Credit Agency - great.gov.uk{% endblock %}
+
+{% block css_layout_class %}uk-export-finance-landing{% endblock %}
+
+{% block content %}
+	<section id="uk-finance-export-hero" class="hero hero-generic uk-export-finance-hero" style="background-image: url({% static 'images/project_finance_hero_image.jpg' %})">
+		<div class="container">
+			<div class="hero-title-compact">
+				<h1 class="heading-hero-generic-compact hero-title" id="hero-heading">
```
[the remark this answers is not in the document]
Comment (erdem) [reply ukef/templates/ukef/landing_page.html]: All lines indented with spaces, did you see as a tab on your editor? @Kenneth-Siu

File: core/sass/ukef-landing-page.scss (hunk 1 above)
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: I am using this CSS class with `.heading`. The class overriding the `font-family` only in that case. 

https://github.com/uktrade/great-domestic-ui/pull/119/files#diff-5a7a91d12b6028baa61a790e47f0ae56R46

File: core/sass/ukef-landing-page.scss (hunk 8 above)
[the remark this answers is not in the document]
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: removed

File: core/sass/ukef-landing-page.scss (hunk 5 above)
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: removed

File: core/sass/ukef-landing-page.scss (hunk 6 above)
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: removed

File: ukef/templates/ukef/landing_page.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (Kenneth-Siu) [reply ukef/templates/ukef/landing_page.html]: Mmm... they seem to be tabs to me. I checked out the branch and they're also tabs. Is your editor doing some behind-the-scenes magic or something, maybe?

[the remark this answers is not in the document]
Comment (erdem) [reply ukef/templates/ukef/landing_page.html]: Weird looks space on my editor but they actually tabs. I'll look into this.

File: core/sass/ukef-landing-page.scss (hunk 1 above)
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: @Kenneth-Siu I replaced this class with `heading-large`.

File: core/sass/ukef-landing-page.scss (hunk 2 above)
Comment (erdem) [reply core/sass/ukef-landing-page.scss]: ok, I replaced to `text` with 'font-medium'.

**Pull request merged**